Sources/SwiftCheck/Gen.swift

```diff
@@ -153,6 +108,59 @@ public struct Gen<A> {
 	}
 }
 
+extension Gen where A: RandomType, A: Comparable {
```

Comment:
Nit: We use spaces around the front and back of colons for type annotations (and the regular style for parameters).  Please clean up the other extensions as well.